### PR TITLE
Add create blank form option to form template selection [MAILPOET-3207]

### PR DESCRIPTION
**Pull request by costasovo**
[MAILPOET-3207]

[MAILPOET-3207]: https://mailpoet.atlassian.net/browse/MAILPOET-3207

### CODE REVIEW

File: assets/css/src/components-form-editor/_template-selection.scss [hunk 1]
```diff
@@ -27,11 +27,19 @@ $mailpoet-form-template-thumbnail-height: 316px;
 }
 
 .template-selection-header {
-  align-items: center;
   background: $color-input-background;
   border-bottom: 1px solid $color-tertiary-light;
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: center;
+  display: grid;
```
Comment (pavel-mailpoet) [inline assets/css/src/components-form-editor/_template-selection.scss:32]: could you please explain why did you change this to a grid? I can see only a single line in there...

Comment (costasovo) [reply assets/css/src/components-form-editor/_template-selection.scss:32]: I added the button so there are 2 items in the top bar. I needed heading to be positioned in centre and button on right. I found out that it is quite tricky to do with flexbox and I found using the grid as the most elegant solution.

Comment (pavel-mailpoet) [reply assets/css/src/components-form-editor/_template-selection.scss:32]: I reckon that grid "trick" deserves a comment in the code, what do you think?

**Review by pavel-mailpoet — DISMISSED** [review]

**Review by veljkho — CHANGES_REQUESTED** [review]
Is it possible to [align the button](https://i.imgur.com/XPalwKo.png) with the template box grid width? Please try, since like this it's too on the right side.

**Review by NeosinneR — APPROVED** [review]

**Review by veljkho — APPROVED** [review]

**Pull request merged**